stdlib/set.gr

```diff
@@ -15,14 +17,24 @@ record Set<k> {
 }
 
 // TODO: This could take an `eq` function to custom comparisons
+/**
+ * Creates a new, empty set with an initial storage size for the given number of elements.
```

Comment:
I’d add that this is just a best guess and doesn’t limit the size of the set which will grow anyway. 

Do we have any recommendations for making a good guess?